package.json

```diff
@@ -607,7 +608,12 @@
         "category": ".NET"
       },
       {
-        "command": "dotnet.restore",
+        "command": "dotnet.restore.project",
+        "title": "Restore Project",
+        "category": ".NET"
+      },
+      {
+        "command": "dotnet.restore.all",
         "title": "Restore Packages",
```

Comment:
Should we rename this to "Restore All Projects" for symmetry with "Restore Project"?

Reply:
Done.